File: runtimes/dart/time.dart
import 'dart:mirrors';
import 'dart:math';
import 'dart:collection';
import 'dart:io';

${SOLUTION}

${UTILITIES}

void main() {
  final start = Stopwatch()..start();
  ${CALL_SOLUTION};
  start.stop();
  print(start.elapsedMilliseconds);
}

File: runtimes/csharp/time.cs
#nullable disable
using System;
using System.Collections.Generic;
using System.Linq;
using System.Diagnostics;

${SOLUTION}

${UTILITIES}

partial class _Runner {
    public static void Main() {
        var sw = Stopwatch.StartNew();
        ${CALL_SOLUTION};
        sw.Stop();
        Console.Write(sw.ElapsedMilliseconds);
    }
}
